util.py

```diff
@@ -11,4 +11,4 @@ def first_group_name(request):
 
 
 def is_staff(request):

```

Comment:
`is_staff(request)` and `has_group(request)` also don't make much sense. 
```suggestion
def is_staff(user):

```